javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  
+    studentName === "" ? console.log("Please add a name!"):undefined;
+
+    if (studentName == "Queen"  || studentName === "queen"){
+        class07Students.push("Queen");
+        return;
+    } else if(class07Students.length >= 6 && studentName !== "Queen"){
+        return "Sorry there is no more space for other students.";
```

Comment:
It is very good that you are experimenting with your code. I appreciate that :)
However, "return" would not log the message in the console. console log is a better choice for this level!